src/hotspot/share/opto/cfgnode.hpp

```diff
@@ -83,6 +83,7 @@ class RegionNode : public Node {
 
 private:
   bool _is_unreachable_region;
+  bool _shim; // inserted by PEA
```

Comment:
Maybe we should write out `Partial Escape Analysis`. Other devs won't know what PEA is.